src/MainWindow.cc

```diff
@@ -135,8 +135,12 @@ QStringList MainWindow::PluginListModel() const
   {
     for (auto const &plugin : path.second)
     {
-      // Remove lib and .so
-      auto pluginName = plugin.substr(3, plugin.find(".") - 3);
+      // TODO(anyone): Move this into gz-plugin to be reusable
+
+      // Remove shared library prefix and shared library suffix
+      auto pluginName = plugin.substr(
+        strlen(SHARED_LIBRARY_PREFIX),
+        plugin.length() - strlen(SHARED_LIBRARY_SUFFIX));
```

Comment:
I see the `.so` extension when testing on ubuntu:

<img width="423" alt="plugin_filter" src="https://github.com/gazebosim/gz-gui/assets/4000684/9a07c365-03d6-4315-8dd4-4cfa46c03b60">

I think this should fix it:

```suggestion
      auto pluginName = plugin.substr(
        strlen(SHARED_LIBRARY_PREFIX),
        plugin.length() - strlen(SHARED_LIBRARY_PREFIX) -
        strlen(SHARED_LIBRARY_SUFFIX));
```

Reply:
Obviously, thanks! I tested on Windows with this change and it still works.